### PR TITLE
Add security md file

### PR DESCRIPTION
**Pull request by berrydenhartog**
# Description

Added security reporting standard

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Security Policy
+
+## Supported Versions
```
Comment (robbertbos) [inline SECURITY.md:3]: @berrydenhartog curious: is this section relevant for our MkDocs setup?

**Review by laurensWe — APPROVED** [review]
Neat!

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+| --------- | ----------------------------------------- |
+| 9.0-10.0  | Releases within the previous months       |
+| 4.0-8.9   | Most recent release                       |
+
+## Reporting a Vulnerability
+
+Please report (suspected) security vulnerabilities to
+**[ai-validatie@minbzk.nl](mailto:ai-validatie@minbzk.nl)**. You will receive a response from
+us within 5 working days. If the issue is confirmed, we will release a patch as soon
+as possible depending on complexity but usually within a few days.
```
Comment (robbertbos) [inline SECURITY.md:18]: Line length:
```suggestion
Please report (suspected) security vulnerabilities to **[ai-validatie@minbzk.nl](mailto:ai-validatie@minbzk.nl)**. You
will receive a response from us within 5 working days. If the issue is confirmed, we will release a patch as soon as
possible depending on complexity but usually within a few days.
```

**Review by robbertbos — COMMENTED** [review]
LGTM

**Pull request merged**